src/scenes/ProgressReports/EditForm/Steps/ExplanationOfProgress/ExplanationOfProgress.tsx

```diff
@@ -169,7 +169,18 @@ export const ExplanationOfProgress: StepComponent = ({ report }) => {
               <EnumField
                 name="reasons"
                 label="Select a reason"
-                options={optionsByGroup[group]}
+                options={() =>
+                  optionsByGroup[group].map((reason: any) => (
+                    <EnumOption<OptionGroup>
+                      name={reason}
+                      variant='radio'
+                      value={reason}
+                      disabled={optionsByGroup[group].deprecated.includes(
```

Comment:
I think this would be
```suggestion
                      disabled={optionsByGroup.deprecated.includes(
```